### PR TITLE
fix(base): make the base class a component

### PR DESCRIPTION
**Pull request by kensodemann**

### CODE REVIEW

File: src/components.d.ts [hunk 1]
```diff
@@ -28,9 +28,40 @@ declare global {
   }
   namespace JSXElements {
       export interface MyNameAttributes extends HTMLAttributes {
```
Comment (kensodemann) [inline src/components.d.ts:30]: All of this crap is generated, ignore it, it isn't important to your issue...

File: src/components/my-name/my-name.tsx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import { Component, Prop } from '@stencil/core';
```
Comment (kensodemann) [inline src/components/my-name/my-name.tsx:1]: Everything from here on down in this PR is important to your issue.

File: src/utils/some-base.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { Component } from '@stencil/core';
+
+@Component({
```
Comment (alepop) [inline src/utils/some-base.ts:3]: is it important that the class has a decorator? I can not inherit components from simple classes?

Comment (kensodemann) [reply src/utils/some-base.ts:3]: Unfortunately, that is the case for now.

**Comment by kensodemann** [comment]
Closing out old things, this qualifies... 😂

**Pull request closed without merging**